content/en/docs/contribute/review/for-approvers.md

```diff
@@ -81,14 +81,14 @@ Prow Command | Role Restrictions | Description
 :------------|:------------------|:-----------
 `/lgtm` | Organization members | Signals that you've finished reviewing a PR and are satisfied with the changes.
 `/approve` | Approvers | Approves a PR for merging.
-`/assign` | Reviewers or Approvers | Assigns a person to review or approve a PR
-`/close` | Reviewers or Approvers | Closes an issue or PR.
+`/assign` | Anyone | Assigns a person to review or approve a PR
+`/close` | Organization members | Closes an issue or PR.
```

Comment:
I think the author of the issue or PR can also `/close`, maybe that's obvious.

Reply:
Yeah, I know the PR author can do this (and the [Prow doc](https://prow.k8s.io/command-help?repo=kubernetes%2Fwebsite) have stated it).

But since the table text is short, maybe ignore it would be better.